build.py

```diff
@@ -1121,8 +1121,9 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu,
 COPY --from=min_container /usr/lib/{libs_arch}-linux-gnu/libnccl.so.2 /usr/lib/{libs_arch}-linux-gnu/libnccl.so.2
 COPY --from=min_container /usr/lib/{libs_arch}-linux-gnu/libcudnn.so.8 /usr/lib/{libs_arch}-linux-gnu/libcudnn.so.8
 
+# patchelf is needed to add deps of libcublasLt.so.11 to libtorch_cuda.so
```

Comment:
Looks like the rebase was not done correctly

You seem to be missing some lines here: https://github.com/triton-inference-server/server/blob/r22.04/build.py#L996-L1001

Please double check if other changes have been missed as well.

Reply:
Hi @CoderHam  

The issue came up because this commit doesn't have correct parent version in `main` branch and during the commit it's conflicts as following: 
```
<<<<<<< HEAD
=======
    for noncore in NONCORE_BACKENDS:
        if noncore in backends:
            df += '''
COPY --chown=1000:1000 --from=tritonserver_build {0}/install/backends backends
'''.format(build_dir)
            break

    # This is required since libcublasLt.so is not present during the build
    # stage of the PyTorch backend
    if not FLAGS.enable_gpu and ('pytorch' in backends):
        df += '''
RUN patchelf --add-needed /usr/local/cuda/lib64/stubs/libcublasLt.so.11 backends/pytorch/libtorch_cuda.so
'''

    if len(repoagents) > 0:
        df += '''
COPY --chown=1000:1000 --from=tritonserver_build {0}/install/repoagents repoagents
'''.format(build_dir)

    mkdir(ddir)
>>>>>>> c213be65... Fix CPU only failure for pytorch backend (#4227)
```
But because we no longer using NONCORE_BACKENDS in our `build.py` it generates build failure.